### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by ajparsons**
Updated FAQ, mostly to reflect changes to numerology, but other minor style and content changes.

Addresses #1522

For after blog post is launched.

### CODE REVIEW

File: www/includes/easyparliament/templates/html/static/help.php [hunk 1]
```diff
@@ -149,24 +122,39 @@
...
+				</p>
 
                 <ul>
                     <li>To search for an <strong>exact phrase</strong>, use quotes (""). For
                     example, to find only documents which contain the exact phrase <a
                     href="/search/?q=%22Hutton+Report%22">"Hutton Report"</a>. Quotes can also
-                    be used to prevent stemming, so searching for &ldquo;horse&rdquo; will omit results for
-                    horses or horsing.
+                    be used to prevent stemming, so searching for &ldquo;refugee&rdquo; will omit results for
+                    refugees or reefuge.
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/help.php:157]: don't think reefuge is a word? If it was meant to be refuge, this isn't a good example, I'm afraid, as refugee stems to Zrefuge, sure, but refuge stems to Zrefug, so you don't get overlapping results (and reefuge stems to Zreefug).

File: www/includes/easyparliament/templates/html/static/help.php [hunk 2]
```diff
@@ -53,30 +53,27 @@
...
                     easier to follow debates by highlighting who&rsquo;s speaking, for example, and allow you to search and link to them too.
+					Additionally, we publish records of how MPs have voted.  
                 </p>
 
                 <p>
-                    We also perform some automated analysis, for example, MPs&rsquo; pages will tell you how many times they have spoken,
-                    and whether that&rsquo;s above or below average.
-                </p>
+                    Voting content is imported from <a href="https://www.publicwhip.org.uk/">Public Whip</a>, a separate project 
+					(<a href="https://www.bairwell.com/">run by Bairwell Ltd PHP and Node.JS development</a>)which also works with scraped parliamentary data.
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/help.php:68]: missing space after end bracket

File: www/includes/easyparliament/templates/html/static/help.php [hunk 3]
```diff
@@ -180,36 +168,36 @@
                     <li>Find results including <strong>at least one</strong> of your search
                     terms by using &ldquo;OR&rdquo;:
                     <a href="/search/?q=education+OR+schools">education OR schools</a>
-                    (we also have XOR if that&rsquo;s useful).
+                    (we also have XOR if that&rsquo;s useful, , this returns a speech if it only matches one of the two results).
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/help.php:171]: double comma should be a colon or semicolon

File: www/includes/easyparliament/templates/html/static/help.php [hunk 4]
```diff
@@ -416,29 +393,32 @@
...
+				</p>
 
                 <h3 class="js-toc-item" id="new-feature">
                     I have an idea for a new feature &ndash; do you want to hear it?
                 </h3>
                 <p>
-                    We&rsquo;d love to &ndash; but please be aware that we&rsquo;re a small team, so we can&rsquo;t always
-                    introduce new features as quickly as we&rsquo;d like to. However:
+                    We&rsquo;d love to – but please be aware that TheyWorkForYou is currently running without dedicated funding or resources,
+					so at the moment we're unlikely to introduce new features. However, when we do have funding,we adopt new ideas on a merit
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/static/help.php:407]: space after comma missing

**Review by dracos — CHANGES_REQUESTED** [review]
Some small typos and I think stemming needs a better example :)

File: www/includes/easyparliament/templates/html/static/help.php (hunk 4 above)
Comment (ajparsons) [reply www/includes/easyparliament/templates/html/static/help.php:407]: I think this is resolved but not showing up in the preview, does it need a special character or literally just the space? https://github.com/mysociety/theyworkforyou/blob/twfy-faq-update/www/includes/easyparliament/templates/html/static/help.php

Comment (dracos) [reply www/includes/easyparliament/templates/html/static/help.php:407]: I was referring to "funding,we" on this line, not the comma at the end of the line above it :)

Comment (ajparsons) [reply www/includes/easyparliament/templates/html/static/help.php:407]: doh.

**Review by dracos — APPROVED** [review]

**Pull request merged**